src/dotnet/CarbonAware.Plugins.BasicJsonPlugin.Tests/CarbonAware.Plugins.BasicJsonPlugin.Tests.csproj

```diff
@@ -9,9 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NUnit" Version="3.12.0" />
```

Comment:
Same comment here with regards not touching potential deprecated code.

Reply:
am not sure I'd call it deprecated, until GFS declares it so, but still same question about the reason for the version change.

Reply:
Hi @dnastrain , to me these are unnecessary changes for this PR (a lot of files extra)that are not related with the main PR. Perhaps on another 'cleanup' one these version changes should be addressed, keeping PR smaller and functional focus.

Reply:
you are likely correct @juzuluag -- am hoping to hear back from the PR squad if the version changes were (a) 'nice to have' or (b) 'required due to fixes/features this PR depends on'